.github/workflows/publish.yaml

```diff
@@ -2,7 +2,7 @@ name: NPM Publish
 
 on:
   push:
-    tags: v*
+    tags: [v*]
```

Comment:
Small tweak: `tags` should be an array. See example: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags